source/Public/Domain/Get-FabricDomain.ps1

```diff
@@ -85,12 +78,14 @@ function Get-FabricDomain {
             return $null
         }
 
+        Write-Message "Step 6:"
```

Comment:
I'd suggest removing this - we don't seem to output other step info - and I think that's not needed to the user as a message

```suggestion

```

Reply:
Done